sale_order_general_discount_fixed/tests/test_sale_order_general_discount_fixed.py

```
@@ -0,0 +1,287 @@
+        )
+
+    def test_single_line_full_discount_assigned(self):
+        form = Form(self.env["sale.order"])
+        form.partner_id = self.partner
+        with form.order_line.new() as line:
+            line.product_id = self.product_100
+            line.product_uom_qty = 1
+        form.general_discount_fixed = 10.0
+        order = form.save()
+
+        line = order.order_line.filtered(lambda l: not l.display_type)
```

Flake8 (7.3.0)

[error] ambiguous variable name 'l'

(E741)